contracts/DocumentStore.sol

```diff
@@ -1,103 +1,80 @@
 pragma solidity ^0.5.10;
 
-import "./Ownable.sol";
-
-contract DocumentStore is Ownable {
-    string public name;
-    string public version = "2.3.0";
-
-    /// A mapping of the document hash to the block number that was issued
-    mapping(bytes32 => uint256) documentIssued;
-    /// A mapping of the hash of the claim being revoked to the revocation block number
-    mapping(bytes32 => uint256) documentRevoked;
-
-    event DocumentIssued(bytes32 indexed document);
-    event DocumentRevoked(bytes32 indexed document);
-
-    constructor(string memory _name) public {
-        name = _name;
-    }
-
-    function issue(bytes32 document) public onlyOwner onlyNotIssued(document) {
-        documentIssued[document] = block.number;
-        emit DocumentIssued(document);
-    }
-
-    function bulkIssue(bytes32[] memory documents) public {
-        for (uint256 i = 0; i < documents.length; i++) {
-            issue(documents[i]);
-        }
-    }
-
-    function getIssuedBlock(bytes32 document)
-        public
-        view
-        onlyIssued(document)
-        returns (uint256)
-    {
-        return documentIssued[document];
-    }
-
-    function isIssued(bytes32 document) public view returns (bool) {
-        return (documentIssued[document] != 0);
-    }
-
-    function isIssuedBefore(bytes32 document, uint256 blockNumber)
-        public
-        view
-        returns (bool)
-    {
-        return
-            documentIssued[document] != 0 && documentIssued[document] <= blockNumber;
+import "@openzeppelin/upgrades/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
```

Comment:
previously i copied over the Ownable.sol file because it was very hard to verify on etherscan if the code changed - possibly a non-issue if we fix the openzep version

Reply:
I can see why it'll be a problem, but I think if we verify each time it should be fine? Will like to see if its an issue down the road.